src/generic_core/metrics.ts

```diff
@@ -25,9 +30,16 @@ export class Metrics {
       num_cohorts: 64, prob_p: 0.5, prob_q: 0.75, prob_f: 0.5,
       flag_oneprr: true
     };
+    var stringMetric = {
+      type: 'string', num_bloombits: 8, num_hashes: 2,
+      num_cohorts: 64, prob_p: 0.5, prob_q: 0.75, prob_f: 0.5,
```

Comment:
@bemasc - we should figure out some initial values for these probs that add a bit less noise so we can extract stuff out earlier. No reason to switch the success/failure ones, but may be worth getting some extra fidelity out of these.